migration-guide.adoc

```diff
@@ -103,6 +103,19 @@ Interestingly, this now also aligns with handling for auto-flush events which al
 The behavior of `noInterceptor()` for `SharedSessionBuilder` and (the new) `SharedStatelessSessionBuilder` was aligned with the preexisting semantics of this method on `SessionBuilder` and `StatelessSessionBuilder`.
 The previous behavior may be recovered by calling `noSessionInterceptorCreation()`.
 
+[[test-util-bytecodeenhanced-engine]]
+=== JUnit `BytecodeEnhancedTestEngine`
+
+With this version, `BytecodeEnhancedTestEngine` will be disabled by default.
+To be able to run the tests annotated with `@BytecodeEnhanced`, this engine has to be explicitly enabled via a system property:
+
+[source, properties]
+----
+hibernate.testing.bytecode.enhancement.extension.engine.enabled=true
+----
```

Comment:
Maybe Maven/Gradle config instead? I know you probably have both somewhere, given you use this in Search as well :)